lib/java/com/google/android/material/slider/res/values/attrs.xml

```diff
@@ -69,6 +69,10 @@
     <attr name="trackColorInactive" format="color" />
     <!-- The height of the track. -->
     <attr name="trackHeight" format="dimension" />
+    <!-- The radius of the active tick. -->
+    <attr name="activeTickRadius" format="dimension" />
+    <!-- The radius of the inactive tick. -->
+    <attr name="inactiveTickRadius" format="dimension" />
```

Comment:
Remember to add this in the public.xml.

Reply:
Please rename these to `tickRadiusActive` and `tickRadiusInactive`. And move them right below the `tickColorXXX` attributes.